2013/01/31 08:01:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:12:35 [error] [exception.CException] exception 'CException' with message 'UsersManager and its behaviors do not have a method or closure named "sendNewPassword".' in D:\Dropbox\sites\YII_1_1_13\base\CComponent.php:265
Stack trace:
#0 D:\Dropbox\sites\planteaters\api\protected\controllers\UsersController.php(83): CComponent->__call('sendNewPassword', Array)
#1 D:\Dropbox\sites\planteaters\api\protected\controllers\UsersController.php(83): UsersManager->sendNewPassword(Object(Users), 'yajemowa4')
#2 [internal function]: UsersController->actionPasswordRecovery('a617d1f7bc0b927...', 'arseysensector@...')
#3 D:\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#4 D:\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#5 D:\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(40): CFilterChain->run()
#8 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#12 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('passwordrecover...')
#14 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('users/passwordr...')
#15 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#16 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/api/json/user/password_recovery/key/a617d1f7bc0b927894c46af600629c6f2cbf320f93f6ee9b0d49366834d172009108496f56ff944ac9af6b22ef9cedcd3eb50b0af6ebdb2f330ff04e848b87bd/email/arseysensector%40gmail.com
---
2013/01/31 08:12:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:12:55 [error] [exception.CException] exception 'CException' with message 'UsersManager and its behaviors do not have a method or closure named "sendNewPassword".' in D:\Dropbox\sites\YII_1_1_13\base\CComponent.php:265
Stack trace:
#0 D:\Dropbox\sites\planteaters\api\protected\controllers\UsersController.php(83): CComponent->__call('sendNewPassword', Array)
#1 D:\Dropbox\sites\planteaters\api\protected\controllers\UsersController.php(83): UsersManager->sendNewPassword(Object(Users), 'bacujogi8')
#2 [internal function]: UsersController->actionPasswordRecovery('a617d1f7bc0b927...', 'arseysensector@...')
#3 D:\Dropbox\sites\YII_1_1_13\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#4 D:\Dropbox\sites\YII_1_1_13\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#5 D:\Dropbox\sites\YII_1_1_13\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(40): CFilterChain->run()
#8 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#12 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('passwordrecover...')
#14 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('users/passwordr...')
#15 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#16 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/api/json/user/password_recovery/key/a617d1f7bc0b927894c46af600629c6f2cbf320f93f6ee9b0d49366834d172009108496f56ff944ac9af6b22ef9cedcd3eb50b0af6ebdb2f330ff04e848b87bd/email/arseysensector%40gmail.com
---
2013/01/31 08:12:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:16:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:18:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:19:15 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Login Required' in D:\Dropbox\sites\YII_1_1_13\web\auth\CWebUser.php:385
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(177): CWebUser->loginRequired()
#1 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#2 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#7 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#9 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#10 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#11 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/api/json/feedbacks
---
2013/01/31 08:20:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "json".' in D:\Dropbox\sites\YII_1_1_13\web\CController.php:483
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CController.php(270): CController->missingAction('json')
#1 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('json')
#2 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/json/user/l...')
#3 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/api/json/user/login
---
2013/01/31 08:21:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:21:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:21:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/01/31 08:23:50 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Login Required' in D:\Dropbox\sites\YII_1_1_13\web\auth\CWebUser.php:385
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(177): CWebUser->loginRequired()
#1 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#2 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#7 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#9 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#10 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#11 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/api/json/feedbacks
---
2013/01/31 08:24:39 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Login Required' in D:\Dropbox\sites\YII_1_1_13\web\auth\CWebUser.php:385
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(177): CWebUser->loginRequired()
#1 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#2 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#7 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#9 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#10 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#11 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/api/json/feedbacks
---
2013/01/31 08:25:11 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Login Required' in D:\Dropbox\sites\YII_1_1_13\web\auth\CWebUser.php:385
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(177): CWebUser->loginRequired()
#1 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#2 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#7 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#9 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#10 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#11 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/api/json/feedbacks
---
2013/01/31 08:25:19 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Login Required' in D:\Dropbox\sites\YII_1_1_13\web\auth\CWebUser.php:385
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(177): CWebUser->loginRequired()
#1 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#2 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#7 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#9 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#10 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#11 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/api/json/feedbacks
---
2013/01/31 08:26:37 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Login Required' in D:\Dropbox\sites\YII_1_1_13\web\auth\CWebUser.php:385
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(177): CWebUser->loginRequired()
#1 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#2 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#7 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#9 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#10 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#11 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/api/json/feedbacks
---
2013/01/31 08:29:06 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Login Required' in D:\Dropbox\sites\YII_1_1_13\web\auth\CWebUser.php:385
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(177): CWebUser->loginRequired()
#1 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#2 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#7 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('list')
#9 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/list')
#10 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#11 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/api/json/feedbacks
---
2013/01/31 08:29:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "json".' in D:\Dropbox\sites\YII_1_1_13\web\CController.php:483
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CController.php(270): CController->missingAction('json')
#1 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('json')
#2 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('api/json/user/p...')
#3 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#4 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/api/json/user/password_recovery
---
2013/01/31 08:32:31 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\Dropbox\sites\YII_1_1_13\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 D:\Dropbox\sites\YII_1_1_13\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 D:\Dropbox\sites\YII_1_1_13\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 D:\Dropbox\sites\YII_1_1_13\web\CController.php(291): CFilterChain->run()
#6 D:\Dropbox\sites\YII_1_1_13\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(282): CController->run('passwordrecover...')
#8 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('users/passwordr...')
#9 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#10 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/api/json/user/password_recovery
---
2013/01/31 08:35:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php:286
Stack trace:
#0 D:\Dropbox\sites\YII_1_1_13\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\Dropbox\sites\YII_1_1_13\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\Dropbox\sites\planteaters\api\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
